kolibri/core/bookmarks/models.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import models
+from morango.models import UUIDField
+
+from kolibri.core.auth.models import FacilityUser
+
+
+class Bookmark(models.Model):
+
+    id = UUIDField(primary_key=True, default=uuid.uuid4)
+    content_id = models.CharField(max_length=50, null=False, blank=False)
```

Comment:
Any reason not to have all of these as UUID fields - this will give another level of validation to ensure the values are correct.